pricing.js

```diff
@@ -1,3 +1,23 @@
+function lifePrice(coverageAmount, product, price) {
```

Comment:
Since we name functions based on what they do, they should be verbs. calculateVolLifePrice is a better function name as it tells us what it is doing. volLifePrice is less obvious as it could be calculating it, discounting it, validating it, and a whole host of other things.